ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/AddExtensionAttributeTest.fs

```diff
@@ -0,0 +1,26 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Features
+
+open JetBrains.ReSharper.FeaturesTestFramework.Intentions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type AddExtensionAttributeTest() =
+    inherit QuickFixTestBase<AddExtensionAttributeFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/addExtensionAttribute"
+
+    [<Test>] member x.``Module``() = x.DoNamedTest()
+    [<Test>] member x.``Type``() = x.DoNamedTest()
+    [<Test>] member x.``Attribute namespace - move``() = x.DoNamedTest()
```

Comment:
The test name looks outdated (though, it'd be great to have this logic with moving the import in future).